config.example.toml

```diff
@@ -27,6 +27,9 @@ hf_token = "<API_KEY>" # Hugging face api token, required if you want to access
 
 [atoma_proxy_service]
 service_bind_address = "0.0.0.0:8081" # This should match the port in the docker-compose.yml
+grafana_url = "" # Grafana URL
```

Comment:
This will panic for local instances of the proxy, I believe

Reply:
why should it?
Just do http://localhost, and you are goot to go